apps/condo/domains/common/components/containers/BaseLayout/BaseLayout.tsx

```
@@ -6,6 +6,8 @@ import React, { CSSProperties, FunctionComponent, ElementType } from 'react'
 
 import { useAIContext } from '@condo/domains/ai/components/AIContext'
 import { useLayoutContext } from '@condo/domains/common/components/LayoutContext'
+import { useFeatureFlags } from '@open-condo/featureflags/FeatureFlagsContext'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)